app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/EmployeeView.java

```diff
@@ -18,7 +19,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         setContentView(R.layout.employee_view);
         String employee = getIntent().getStringExtra(EMPLOYEE_NAME);
         if(employee != null){
-            // call a function
+            if (employee.equals("Rusi is amazing") showRusi();
```

Comment:
@rusili56 This obviously doesn't compile. There are 3 open parenthesis and 2 closing ones